logstash-filter-alter.gemspec

```diff
@@ -20,7 +20,7 @@ Gem::Specification.new do |s|
   s.metadata = { "logstash_plugin" => "true", "logstash_group" => "filter" }
 
   # Gem dependencies
-  s.add_runtime_dependency "logstash-core-plugin-api", "~> 1.0"
+  s.add_runtime_dependency "logstash-core-plugin-api", "~> 2.0"
```

Comment:
The current pattern is to use `s.add_runtime_dependency "logstash-core-plugin-api", ">= 1.60", "<= 2.99"` to keep support for LS 2.4.x && 5.x